### PR TITLE
VS 2015, appveyor artifacts

### PR DESCRIPTION
**Pull request by chcg**
- renamed VS project dirs from 2010 to 2015
- updated VS projects to visual studio version 2015
- appveyor publish release artifacts

- added postbuild events for the release builds
- create zip artifacts for release builds, which needs to be further restricted to platformtoolset v140_xp or v120_xp
- added deploy template for automatic github releases

@bruderstein : Could you have a look at this. Is this the direction you intended to go? What should be used for deployment ? The v140_xp or v120_xp build output? See e.g. https://ci.appveyor.com/project/bruderstein/npppluginmanager/build/1.4.3.19/job/3ii0f6be8h33vcty/artifacts The zip file PluginManager__x64.zip is hopefully expanded in case of a tag build to PluginManager_"TagName"_x64.zip. Similar for PluginManager__UNI.zip. For the naming I took the current schema as template.

### CODE REVIEW

File: pluginManager/projects/2015/PluginManager.vcxproj.filters [hunk 1]
```diff
@@ -123,7 +123,6 @@
     </ClCompile>
   </ItemGroup>
   <ItemGroup>
-    <Image Include="..\..\res\nbc_logo.png" />
     <Image Include="..\..\res\nbc_logo.bmp">
```
Comment (chcg) [inline pluginManager/projects/2015/PluginManager.vcxproj.filters]: Just the bmp is available in git, is this intended? VS automatically removes the unkown image file from the filters.

**Comment by bruderstein** [comment]
If you do the following, you'll have a clean copy of this:

`git remote add upstream git@github.com:bruderstein/nppPluginManager.git`  (you may have already done something similar, if you have a remote name for my repo, then you can skip this obviously, and use the right name in the next steps)

 `git fetch upstream`
`git rebase upstream/master`
`git push origin --force-with-lease`

Then your branch will be updated, and clean.

**Comment by bruderstein** [comment]
The size of the binary is a pretty strong argument. A bit of googling doesn't reveal much, but in the past there's some options to try and fix it.

I'm leaning back to 2013 unless we can find out what's causing the size increase - what do you thing?

**Comment by chcg** [comment]
Both binaries are build from the same solution, just the platform toolset is different, so I was a little bit stunning about that big file size impact. Maybe some tests show an option to decrease the size again with VS2015. 

For now VS2013 seems also more appealing to me. 

I will rework this PR then.

**Comment by chcg** [comment]
For cleanup see: https://github.com/bruderstein/nppPluginManager/pull/37
which is prepared for publishing the build with v120_xp aka VS2013 versions of the build

**Pull request closed without merging**